velox/type/DecimalUtil.h

```diff
@@ -17,23 +17,14 @@
 #pragma once
 
 #include <string>
+#include "velox/common/base/CheckedArithmetic.h"
 #include "velox/common/base/Exceptions.h"
 #include "velox/type/Type.h"
 #include "velox/type/UnscaledLongDecimal.h"
 #include "velox/type/UnscaledShortDecimal.h"
 
 namespace facebook::velox {
 
-#if defined(__has_feature)
```

Comment:
@karteekmurthys don't remove this.

Reply:
FYI I found the root cause of the sanitizer bug.  We can use the plain vanilla `*` now and `mul` is no longer needed.

Reply:
Thanks @Yuhta

Reply:
@Yuhta Is this resolved for `+` also?

Reply:
`+` should be ok all the time

Reply:
Thanks @Yuhta. Is the fix already merged?

Reply:
Just a moment, seems my fix has some conflicts with some internal rust code.  Still need some time to find a solution.

Reply:
We still need to keep it because of the conflicts with Rust